vpc/service/branch_enis.go

```diff
@@ -1042,7 +1042,8 @@ type actionWorker struct {
 
 	pendingState string
 
-	readyCh chan struct{}
+	readyCh       chan struct{}
+	readyChClosed bool
```

Comment:
i'm assuming there are no concurrency concerns that would require a lock here?

Reply:
Shouldn't.